.github/workflows/cicd_test_dev.yml

```diff
@@ -0,0 +1,23 @@
+name: Trigger CICD Test From Dev
+
+on:
+  workflow_run:
+    workflows: ["CD"]
+    branches: ["dev"]
+    types: ["completed"]
+
+jobs:
+  TriggerCICDTestFromDev:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Trigger TeamsFx-CICD-Test
+        env:
+            TEAMSFX_CICD_TOKEN: ${{ secrets.TEAMSFX_CICD_TOKEN }}
+        run: |
+            curl \
+                -u IvanJobs:${TEAMSFX_CICD_TOKEN} \
```

Comment:
-u :${{ secrets.GITHUB_TOKEN }}
this token is managed by Wei's team centrally.

Reply:
Thanks.